HorizonEngine/src/HorizonEngine/Renderer/Sprite.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
+
+#ifndef HZN_SPRITE_H
+#define HZN_SPRITE_H
+
+#include "Texture.h"
+
+namespace Hzn
+{
+	/// <summary>
```

Comment:
I like the use of documentation throughout this PR, makes it easier to understand, cheers!